.agents/skills/e2e-container/scripts/start-envtest.sh

```
@@ -0,0 +1,47 @@
+  fi
+  [ -f /tmp/envtest-kubeconfig ] && break
+  sleep 1
+done
+if [ ! -f /tmp/envtest-kubeconfig ]; then
+  echo "ERROR: timed out waiting for envtest kubeconfig after 60s"
+  cleanup
+  exit 1
+fi
+
+echo "Waiting for API server readiness..."
+for i in $(seq 1 15); do
```

Shellcheck (0.11.0)

[warning] i appears unused. Verify use (or export if used externally).

(SC2034)